builtin/providers/aws/resource_aws_route53_record.go

```diff
@@ -407,12 +407,13 @@ func resourceAwsRoute53RecordBuildSet(d *schema.ResourceData, zoneName string) (
 		rec.HealthCheckId = aws.String(v.(string))
 	}
 
-	if v, ok := d.GetOk("weight"); ok {
-		rec.Weight = aws.Int64(int64(v.(int)))
-	}
+	//if v, ok := d.GetOk("weight"); ok {
+	//	rec.Weight = aws.Int64(int64(v.(int)))
+	//}
```

Comment:
I think it's better to just remove code that's no longer needed, rather than leaving it commented. If anyone wants to get this code back it'll be saved permanently in the git history, and anyone reading the latest code will not be distracted by wondering what this was here for.